plugins/nodebb-plugin-caiz/static/community-edit.js

```
@@ -1421,6 +1490,8 @@ const updateCategoryOrder = () => {
 // Make functions globally available for onclick handlers
 window.editCategory = editCategory;
 window.deleteCategory = deleteCategory;
+window.changeMemberRole = changeMemberRole;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)